### PR TITLE
Url update2

### PR DESCRIPTION
**Pull request by ashley-o0o**
More links fixed. 16-18, 20-29

Progress toward: https://github.com/patternfly/patternfly-org/issues/3437

Closes https://github.com/patternfly/patternfly-org/issues/3437

### CODE REVIEW

File: beta-component-promotion/promotion-2022.14/beta-promotion-candidates.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ If you don't want multiple alerts to stack and fill the screen until the user di
 ![alert group overflow](./img/alert-group-overflow.png)
 
 ### [Calendar month](https://www.patternfly.org/v4/components/calendar-month)
-A calendar month component allows users to select and navigate between days, months and/or years. This component is usually used with the date picker component, to display date options and selections.
+A calendar month com. This component is usually used with the date picker component, to display date options and selections.
```
Comment (nicolethoen) [inline beta-component-promotion/promotion-2022.14/beta-promotion-candidates.md:10]: I think this is a typo

**Comment by ashley-o0o** [comment]
**What:** Closes [#3437](https://github.com/patternfly/patternfly-org/issues/3437)

File: beta-component-promotion/promotion-2022.14/beta-promotion-candidates.md (hunk 1 above)
Comment (ashley-o0o) [reply beta-component-promotion/promotion-2022.14/beta-promotion-candidates.md:10]: Fixed later here https://github.com/patternfly/patternfly-org/pull/3557

**Comment by nicolethoen** [comment]
This PR needs to be rebased to resolve conflicts, too. I see that this PR has the incorrect information about next/deprecated components in the component summaries. I think you've fixed that in other PRs, but i'll double check after you rebase.

**Comment by nicolethoen** [comment]
This looks like it needs to be rebased again

**Comment by nicolethoen** [comment]
Looks like these changes were pulled into other PRs

**Pull request closed without merging**